File: PROYECTO/PSport-APP/PSport/Model/DTOs/TorneoDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PSport.Model.DTOs
{
    public class TorneoDTO2
    {
        public Guid? id { get; set; }
        public string nombre { get; set; }
        public int deporte { get; set; }
        public int tipoTorneo { get; set; }
        public string estado { get; set; }
        public string fechaInicio { get; set; }
        public string fechaFin { get; set; }
        public List<dynamic>? partidos { get; set; }
        public List<dynamic>? jugadoresTorneo { get; set; }
        public List<dynamic>? equiposTorneo { get; set; }
    }

    public class TorneoDTO
    {
        public Guid? id { get; set; }
        public string nombre { get; set; }
        public string deporte { get; set; }
        public string tipoTorneo { get; set; }
        public string estado { get; set; }
        public string fechaInicio { get; set; }
        public string fechaFin { get; set; }
        public List<dynamic>? partidos { get; set; }
        public List<dynamic>? jugadoresTorneo { get; set; }
        public List<dynamic>? equiposTorneo { get; set; }
    }
}

File: PROYECTO/old/PP-Sport-API-/PPSportAPI/src/main/kotlin/es/sergigavi/PPSportAPI/MODEL/DTO/TorneoDTO.kt
package es.sergigavi.PPSportAPI.MODEL.DTO

import es.sergigavi.PPSportAPI.MODEL.*
import java.time.LocalDateTime
import java.util.*

data class TorneoDTO(

    val id: UUID,
    val deporte: Deporte,
    var nombre: String,
    val tipoTorneo: TipoTorneo,
    var estado: Estado,
    val fechaInicio: LocalDateTime,
    val fechaFin: LocalDateTime,
    val partidos: MutableSet<PartidoDTO>,
    val equipoTorneos: MutableSet<EquipoTorneoDTO>,
    val jugadores: MutableSet<JugadorTorneoTDTO>
    )
